packages/react-components/src/templates/GroupProfileAbout.tsx

```diff
@@ -59,9 +57,7 @@ const GroupProfileAbout: React.FC<GroupProfileAboutProps> = ({
         active={active}
       />
       <div id={membersSectionId}>
-        {active ? (
-          <GroupMembersSection teams={teams} leaders={leaders} />
-        ) : (
+        {!active && (
```

Comment:
This means that when the team is active is not going to appear neither the leaders or teams while it's not replaced by the new designs. Is this the idea? I don't remember if we addressed this in refinement and it's not that clear in the ticket 😛

Reply:
I checked with Tiffany and leaders and teams should now appear for both active and inactive groups. I'll apply it now. Thanks for spotting that @gabiayako !😃